webapp/platform/types/src/theme.ts

```diff
@@ -0,0 +1,35 @@
+export type ThemeKey = 'denim' | 'sapphire' | 'quartz' | 'indigo' | 'onyx';
+
+export type LegacyThemeType = 'Mattermost' | 'Organization' | 'Mattermost Dark' | 'Windows Dark';
+
+export type ThemeType = 'Denim' | 'Sapphire' | 'Quartz' | 'Indigo' | 'Onyx';
+
+export type Theme = {
+    [key: string]: string | undefined;
+    type?: ThemeType | 'custom';
+    sidebarBg: string;
+    sidebarText: string;
+    sidebarUnreadText: string;
+    sidebarTextHoverBg: string;
+    sidebarTextActiveBorder: string;
+    sidebarTextActiveColor: string;
+    sidebarHeaderBg: string;
+    sidebarTeamBarBg: string;
+    sidebarHeaderTextColor: string;
+    onlineIndicator: string;
+    awayIndicator: string;
+    dndIndicator: string;
+    mentionBg: string;
+    mentionBj: string;
```

Comment:
Similarly, could you remove this one as well? It's the result of an ancient typo so I don't want to copy it over to the new package

Reply:
<img width="524" alt="Screenshot 2023-05-03 at 11 49 20 AM" src="https://user-images.githubusercontent.com/16203333/235844351-870d6631-eb8e-4c75-9e0d-5a72570a62cb.png">
What about this?

Reply:
Although I am in favour of removing it in 8.0 or at-least depreciation warning

Reply:
For that one, I think we can have it use `(theme as any).mentionBj`. It's kind of ugly, but it's the only place that the old field name should ever be needed